charts/cloudnative-pg/values.yaml

```diff
@@ -29,6 +29,9 @@ imagePullSecrets: []
 nameOverride: ""
 fullnameOverride: ""
 
+hostNetwork: false
+dnsPolicy: "ClusterFirst"
```

Comment:
```suggestion
dnsPolicy: ""
```

Reply:
As @itay-grudev properly mention this should be "Default" since that's the proper default an empty string is not on the list of possible options

Reply:
On the other hands, these two values should be set only if there's any value set, having a default it's like mandatory, in platforms like OpenShift this hostNetwork value and dnsPolicy may need to be empty

Reply:
@sxd, the default is actually `ClusterFirst` as per https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-s-dns-policy.

> "Default" is not the default DNS policy. If dnsPolicy is not explicitly specified, then "ClusterFirst" is used.

Reply:
As I said, better to not set any default value, only set if it's set by the user

Reply:
Changed the default value of `dnsPolicy` to an empty string: https://github.com/cloudnative-pg/charts/pull/324/commits/825684865ef45bc47a92169673796b45e4a92a38

Reply:
> On the other hands, these two values should be set only if there's any value set, having a default it's like mandatory, in platforms like OpenShift this hostNetwork value and dnsPolicy may not need to be set

That's why I suggested we use `""` for `dnsPolicy` in the `values.yaml` file. That way, the field won't get rendered in the deployment template.

`hostNetwork` is already `false` by default in the `values.yaml`, so it won't get rendered as it is wrapped in a conditional.